tensorflow/compiler/mlir/tosa/tests/tf-to-tosa-pipeline.mlir

```diff
@@ -105,13 +105,13 @@ func @test_relu6(%arg0: tensor<13x21x3xf32>) -> tensor<13x21x3xf32> {
 // -----
 
 // CHECK-LABEL: test_leaky_relu
-// CHECK: tosa.const
-// CHECK: tosa.const
-// CHECK: tosa.reshape
-// CHECK: tosa.mul
-// CHECK: tosa.reshape
-// CHECK: tosa.greater_equal
-// CHECK: tosa.select
+// CHECK-DAG: %[[VAR0:.*]] = "tosa.const"() {value = dense<0.000000e+00> : tensor<f32>}
+// CHECK-DAG: %[[VAR1:.*]] = "tosa.const"() {value = dense<5.000000e-01> : tensor<f32>}
+// CHECK: %[[VAR2:.*]] = "tosa.reshape"(%[[VAR1]]) {new_shape = [1, 1]}
+// CHECK: %[[VAR3:.*]] = "tosa.mul"(%arg0, %[[VAR2]]) {shift = 0 : i32}
+// CHECK: %[[VAR4:.*]] = "tosa.reshape"(%[[VAR0]]) {new_shape = [1, 1]}
+// CHECK: %[[VAR5:.*]] = "tosa.greater_equal"(%arg0, %[[VAR4]])
+// CHECK: %[[VAR6:.*]] = "tosa.select"(%[[VAR5]], %arg0, %[[VAR3]])
```

Comment:
Comment regard CHECK vs CHECK-DAG. CHECK-DAG allows different ordering of operations, so it should be preferred in most cases, though the root nodes on the compute graph need to be CHECK statements. In this case, only the "tosa.select" should be a CHECK, the rest should be CHECK-DAGs.

This is needed a many of the checks below.

Reply:
Thanks. Just apply the fix to all other tests in commit#2